docs/source/contributing.rst

```diff
@@ -0,0 +1,187 @@
+Contributing to Kordac
+#######################################
+
+The git repository for Kordac can be found here_.
+
+.. _here: https://github.com/uccser/kordac
+
+
+Issue Reporting and Bug Fixes
+=======================================
+If you come across a bug in Kordac, please `report it`_ on the repo.
+
+<Bug fix? Add tests.>
+
+.. _report it: https://github.com/uccser/kordac/issues
+
+The Code Base
+=======================================
+If you would like to contribute to Kordac, fork the repository.
+
+Overview
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before reading this section, make sure you have read `how to use Kordac`_ (or even better, have already used Kordac!).
+
+.. _how to use Kordac: http://kordac.readthedocs.io/en/develop/usage.html
+
+
+There are several parts and terms of Kordac to become familiar with:
+
+- **Tag**
+    This refers to the custom markdown syntax that Kordac processes.
+    
+    For example:
+    
+    .. code-block:: none
+
+      {comment this will be removed by the converter}
+
+      {image file-path='img/totally-real-image.png' alt='process me'}
+    
+    are examples of the ``comment`` and ``image`` tags in Kordac.  
+
+- ``Kordac()``
+	The convertor object itself. This is what a user will use to create a Kordac converter, and what is used to define a custom processor list, custom html templates and custom Markdown Extensions to use.
+
+- ``KordacResult()``
+    The object returned by ``Kordac()`` containing:
+    	- Converted html string
+    	- Title
+    	- Required files (images, interactives, scratch images, page scripts)
+    	- Heading tree
+    	- Required glossary terms
+
+- ``KordacExtension()``
+    Inherits the ``Extension`` class from Markdown.
+    This class is the main class of the project. It loads all the processor information, loads the template files and clears and populates the attributes to be returned by the ``KordacResult`` object.
+
+- ``Processors/``
```

Comment:
Having folders listed after terms and Python objects is slightly confusing. Possibly define all terms first (define processor there), then have a new section for folders.